internal/template-bundle/bundle_test.go

```diff
@@ -1,48 +1,61 @@
 package template_bundle
 
 import (
+	"runtime"
 	"testing"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
+func retrieveSuffix() string {
+	switch runtime.GOARCH {
+	case "amd64":
+		return ""
```

Comment:
Same as default case, might drop that? :)

Reply:
Done